sync-deps/src/main/java/com/spotify/syncdeps/model/MavenCoords.java

```diff
@@ -42,6 +43,9 @@
   @JsonProperty("artifactId")
   public abstract String artifactId();
 
+  @JsonProperty("classifier")
+  public abstract Optional<String> classifier();
```

Comment:
I think this might have to be `@Nullable` for Jackson to work with it (not that we ever serialize this to JSON anyway...)